charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.8.6
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.50.0
+version: 5.51.0
```

Comment:
I think it's `fixed` PR instead of `feat`, so Maybe bumping patch version is ok.

Reply:
@metajiji 
What about this? 👀

Reply:
Sorry, i agree yes this is patch. Done!